src/main/kotlin/org/rust/lang/core/psi/ext/RsMetaItem.kt

```diff
@@ -17,10 +17,7 @@ import org.rust.lang.core.resolve.ref.RsReference
 import org.rust.lang.core.resolve.ref.deriveReference
 import org.rust.lang.core.stubs.RsMetaItemStub
 
-val RsMetaItem.name: String? get() {
-    val stub = greenStub
-    return if (stub != null) stub.name else identifier?.unescapedText
-}
+val RsMetaItem.name: String? get() = greenStub?.name ?: identifier?.unescapedText
```

Comment:
This change causes test failures. Due to `RsMetaItemStub.name` is nullable, these constructions are not equivalent.

Reply:
Oh really, fixed